pkg/ext-proc/main.go

```diff
@@ -124,7 +106,7 @@ func main() {
 		},
 		Record: mgr.GetEventRecorderFor("InferencePool"),
 	}).SetupWithManager(mgr); err != nil {
-		klog.Error(err, "Error setting up InferencePoolReconciler")
+		klog.Fatalf("Failed setting up InferencePoolReconciler: %v", err)
```

Comment:
Note the switch to `Fatalf` in several places where an error is critical to the extension's startup.